addons/website/models/website.py

```diff
@@ -471,9 +473,12 @@ def set_images(images):
         tour_asset_id.copy({'key': tour_asset_id.key, 'website_id': website.id, 'active': True})
 
         # logo was generated as base64 url
-        logo = kwargs.get('logo')
-        if logo:
-            website.logo = logo.split(',', 1)[1]
+        logo_attachment_id = kwargs.get('logo_attachment_id')
+        if logo_attachment_id:
+            attachment = self.env['ir.attachment'].browse(logo_attachment_id)
+            website.logo = attachment.datas
+            domain = [('res_id', '=', website.id), ('res_model', '=', 'website'), ('name', '=', 'temporary_logo')]
+            self.env['ir.attachment'].search(domain).unlink()
```

Comment:
No other way to "reattach" the attachment to the website logo one? instead of recreate + delete. cc @JKE-be

Reply:
What if you just set `res_field` to `logo`? Won't the ORM read that attachment when reading website.logo ?

Reply:
Yes I think it should work

Reply:
> What if you just set `res_field` to `logo`? Won't the ORM read that attachment when reading website.logo ?

Indeed, I probably asked that question 10 times already and I never can remember the answer. Hopefully I will this time :grimacing:

Reply:
Same, reading the code is seems like Binary fields do get their value from the `read()` method of the Binary class, just doing a search for the required `res_id`, `res_name` and `res_field`.